go.mod

```diff
@@ -66,6 +66,7 @@ require (
 	k8s.io/kubectl v0.20.2
 	k8s.io/mount-utils v0.20.4
 	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
+	k8s.io/kubelet v0.20.5
```

Comment:
I don't think we need this dependency now with the templating approach

Reply:
:+1: cleaned up